src/main/java/io/cdap/plugin/googleads/source/batch/GoogleAdsBatchSourceConfig.java

```diff
@@ -155,36 +158,70 @@ protected void validateAuthorisation(FailureCollector failureCollector, GoogleAd
   }
 
   protected void validateReportTypeAndFields(FailureCollector failureCollector, GoogleAdsHelper googleAdsHelper) {
-    if (containsMacro(REPORT_TYPE)){
+    if (containsMacro(REPORT_TYPE)) {
       return;
     }
     ReportDefinitionReportType reportDefinitionReportType = null;
     try {
       reportDefinitionReportType = getReportType();
     } catch (IllegalArgumentException ex) {
-      failureCollector.addFailure("Invalid reportDefinitionReportType", "Enter valid reportDefinitionReportType").withConfigProperty(REPORT_TYPE);
+      failureCollector.addFailure("Invalid reportDefinitionReportType",
```

Comment:
This message should be readable to user. Something they can understand.
Like reportType '%s' is not a valid report type. Or something like that I am not sure what's best in your case.